Parsers/ASimRegistryEvent/Parsers/vimRegistryEventTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,137 @@
+Parser:
+  Title: Registry Event ASIM Parser
+  Version: '0.1.0'
+  LastUpdated: Jul 17, 2023
+Product:
+  Name: Trend Micro Vision One
+Normalization:
+  Schema: RegistryEvent
+  Version: '0.1.2'
+References:
+- Title: ASIM Registry Schema
+  Link: https://aka.ms/ASimRegistryEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Trend Micro Vision One documentation
+  Link: 
+    https://docs.trendmicro.com/en-us/enterprise/trend-vision-one/xdr-part/search-app/data-mapping-intro/data-mapping-detecti.aspx
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques-Pipeline/paths/~1v3.0~1oat~1dataPipelines~1%7Bid%7D~1packages~1%7BpackageId%7D/get
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques/paths/~1v3.0~1oat~1detections/get
+Description: |
+  This ASIM parser supports normalizing Trend Micro Vision One logs to the ASIM Registry Event normalized schema. Trend Micro Vision One events are captured through Trend Vision One data connector which ingests XDR logs into Microsoft Sentinel through the Trend Vision One API.
+ParserName: vimRegistryEventTrendMicroVisionOne
+EquivalentBuiltInParser: _Im_RegistryEvent_TrendMicroVisionOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventTypeLookup = datatable(detail_eventSubId_s: string, EventType: string)[
+    "TELEMETRY_REGISTRY_CREATE", "RegistryKeyCreated",
+    "TELEMETRY_REGISTRY_SET", "RegistryValueSet",
+    "TELEMETRY_REGISTRY_DELETE", "RegistryKeyDeleted",
+    "TELEMETRY_REGISTRY_RENAME", "RegistryKeyRenamed"
```

Comment:
Only one type exists in sample logs. Hope you have tested with other EventTypes
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/5c5ef05f-011a-4dff-b7f8-9413853cd7cd)

Reply:
We don't have the other type available in any sample as well as live data. But whatever used/mentioned in the parser for other type is from the portal and its official document.

Reply:
Did you use sample data or built environment for the logs used in this parser?
In general, If we are building environment, we should try to use all possible activities that our schema supports. Would make our parser better.

Reply:
We have used logs generated via the environment. 
We have checked with our IT team, but other logs are unavailable in the environment. Hence we have only one type of logs available in our environment as well as in sample data.